packages/teams-js/src/internal/globalVars.ts

```diff
@@ -3,10 +3,10 @@ export class GlobalVars {
   public static initializeCalled = false;
   public static initializeCompleted = false;
   public static additionalValidOrigins: string[] = [];
-  public static initializePromise: Promise<void>;
+  public static initializePromise: Promise<void> | null = null;
   public static isFramelessWindow = false;
-  public static frameContext: FrameContexts;
-  public static hostClientType: string;
+  public static frameContext: FrameContexts | undefined;
+  public static hostClientType: string | undefined;
```

Comment:
Since these two variables were previously declared but not initialized, they were already being set to `undefined` by default. This change updates the type to reflect the behavior that was already occurring.